### ISSUE TITLE
Cannot install .net 6.0 on Windows 10

### ISSUE THREAD

**Issue by stellarleon** [dotnet/core#7766]
Problem encountered on https://dotnet.microsoft.com/en-us/learn/dotnet/hello-world-tutorial/install
Operating System: windows
 
Hello, I verified I have x64 bit, but I attempted to double click and install .net 6.0 on my PC and when I do, nothing happens at all. There's no prompt to install at all. I've restarted my computer and I've tried searching around but I don't see any answers to this particular issue.

**Comment by mairaw** [comment]
What happens when you click on the button to download? 
Currently, it should take you to https://download.visualstudio.microsoft.com/download/pr/9a1d2e89-d785-4493-aaf3-73864627a1ea/245bdfaa9c46b87acfb2afbd10ecb0d0/dotnet-sdk-6.0.400-win-x64.exe

Nothing? Did you check your downloads folder to see if the installer is there?

**Comment by stellarleon** [comment]
> 

It downloads just fine, it shows up in my downloads folder. But when I double click to begin installation, nothing actually happens. I attached a screenshot.
![image](https://user-images.githubusercontent.com/112724264/188218923-eaa18473-a8a1-4931-9a8d-391328a97a8a.png)


I just followed the link that you shared, however, just to try again and it says "failed - network error." Not sure if this is a new issue or not.
